lib/ansible/modules/cloud/amazon/ec2_ami_copy.py

```diff
@@ -24,7 +24,7 @@
 module: ec2_ami_copy
 short_description: copies AMI between AWS regions, return new image id
 description:
-    - Copies AMI from a source region to a destination region. This module has a dependency on python-boto >= 2.5
+    - Copies AMI from a source region to a destination region. This module has a dependency on boto3
```

Comment:
changes like this should be clear about version:

'Since version 2.3 this module depends on boto3'